pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -0,0 +1,763 @@
+	OuterVolumeSpecName string
+
+	// Pod to mount the volume to. Used to create NewMounter.
+	Pod *api.Pod
+
+	// PluginIsAttachable indicates that the plugin for this volume implements
+	// the volume.Attacher interface
+	PluginIsAttachable bool
+}
+
+// AttachedVolume represents a volume that is attached to a node.
+type AttachedVolume struct {
+	// VolumeName is the unique identifier for the volume that is attached.
+	VolumeName api.UniqueVolumeName
+
+	// VolumeSpec is the volume spec containing the specification for the
+	// volume that is attached.
+	VolumeSpec *volume.Spec
+
+	// NodeName is the identifier for the node that the volume is attached to.
+	NodeName string
+
+	// PluginIsAttachable indicates that the plugin for this volume implements
+	// the volume.Attacher interface
+	PluginIsAttachable bool
+}
+
+// MountedVolume represents a volume that has successfully been mounted to a pod.
+type MountedVolume struct {
+	// PodName is the unique identifier of the pod mounted to.
+	PodName volumetypes.UniquePodName
+
+	// VolumeName is the unique identifier of the volume mounted to the pod.
+	VolumeName api.UniqueVolumeName
+
+	// InnerVolumeSpecName is the volume.Spec.Name() of the volume. If the
+	// volume was referenced through a persistent volume claims, this contains
+	// the name of the bound persistent volume object.
+	// It is the name that plugins use in their pod mount path, i.e.
+	// /var/lib/kubelet/pods/{podUID}/volumes/{escapeQualifiedPluginName}/{innerVolumeSpecName}/
+	// PVC example,
+	//   apiVersion: v1
+	//   kind: PersistentVolume
+	//   metadata:
+	//     name: pv0003				<- InnerVolumeSpecName
+	//   spec:
+	//     capacity:
+	//       storage: 5Gi
+	//     accessModes:
+	//       - ReadWriteOnce
+	//     persistentVolumeReclaimPolicy: Recycle
+	//     nfs:
+	//       path: /tmp
+	//       server: 172.17.0.2
+	// Non-PVC example:
+	//   apiVersion: v1
+	//   kind: Pod
+	//   metadata:
+	//     name: test-pd
+	//   spec:
+	//     containers:
+	//     - image: gcr.io/google_containers/test-webserver
+	//     	 name: test-container
+	//     	 volumeMounts:
+	//     	 - mountPath: /test-pd
+	//     	   name: test-volume
+	//     volumes:
+	//     - name: test-volume			<- InnerVolumeSpecName
+	//     	 gcePersistentDisk:
+	//     	   pdName: my-data-disk
+	//     	   fsType: ext4
+	InnerVolumeSpecName string
+
+	// outerVolumeSpecName is the podSpec.Volume[x].Name of the volume. If the
+	// volume was referenced through a persistent volume claim, this contains
+	// the podSpec.Volume[x].Name of the persistent volume claim.
+	// PVC example:
+	//   kind: Pod
+	//   apiVersion: v1
+	//   metadata:
+	//     name: mypod
+	//   spec:
+	//     containers:
+	//       - name: myfrontend
+	//         image: dockerfile/nginx
+	//         volumeMounts:
+	//         - mountPath: "/var/www/html"
+	//           name: mypd
+	//     volumes:
+	//       - name: mypd				<- OuterVolumeSpecName
```

Comment:
No. `OuterVolumeSpecName` is `podSpec.Volume[x].Name` which in this case is `mypd`